### DISCUSSION TITLE
How do I delete all my rooms?

### DISCUSSION THREAD

**Discussion by ninest** [liveblocks/liveblocks#426] [Help] [answered]
I just learned about the "Live" and "Test" environments. In my previous testing, I ended up creating quite a few Live rooms, and I would like to delete them all at once. Is there a faster way to do this? It seems I currently I have to click on each of them, click delete, then confirm. I'd like to delete them because I created rooms out of random IDs for testing an application, but I no longer need them.

(Delete room storage, not delete room)

**Comment by vinodsantharam** [comment, marked as answer]
Hi @ninest! Currently, you can't delete rooms. But you can delete the room's storage through the dashboard or the API. Deleting a room is a feature that is planned for the future. 
We can delete manually your rooms if you want. Let us know!

**Reply by ninest** [reply]
Thanks for the quick response! I ended up using the REST API, which was easy to use. Thanks for the quick response :)

Edit: Yes, I meant delete room storage
